dotnetv3/cross-services/Anniversary_SNS/CreateTable/CreateTable/CreateTable.cs

```diff
@@ -0,0 +1,197 @@
+                KeySchema = keySchema,
+                ProvisionedThroughput = throughput
+            });
+
+            // It can take time to create a table. Wait until the
+            // table status is Active.
+            var tableCreated = await WaitTilTableCreated(client, tableName, response.TableDescription);
+
+            return tableCreated;
+        }
+
+        /// <summary>
+        /// This method waits until the status of the table is "ACTIVE"
+        /// to show that the table was created and is ready for us to
+        /// add sample data.
+        /// </summary>
+        /// <param name="client">The DynamoDB client object used to call
+        /// DescribeTableAsync.</param>
+        /// <param name="tableName">A string representing the name of the table
+        /// that was created.</param>
+        /// <param name="tableDescription">A TableDescription object used
+        /// to check the status of the new table.</param>
+        /// <returns></returns>
+        public static async Task<bool> WaitTilTableCreated(
+            IAmazonDynamoDB client,
+            string tableName,
+            TableDescription tableDescription)
+        {
+            DescribeTableResponse resp = new DescribeTableResponse();
+            string status = tableDescription.TableStatus;
+
+            // One second.
+            Int32 sleepDuration = 1000;
+
+            // Wait for a status of "ACTIVE" to be returned.
+            while ((status != "ACTIVE") && (sleepDuration < 10000)) // Don't wait more than 10 seconds.
+            {
+                System.Threading.Thread.Sleep(sleepDuration);
+
+                resp = await client.DescribeTableAsync(new DescribeTableRequest
+                {
+                    TableName = tableName
+                });
+
+                status = resp.Table.TableStatus;
+
+                sleepDuration *= 2;
+            }
+
+            return (status == "ACTIVE");
+        }
+
+        /// <summary>
+        /// Creates a set of three items and adds each to an Amazon DynamoDB
+        /// table. The first item in the list has its date set to one year
+        /// before the current date to ensure that executing the Lambda
+        /// function will find an employee to whom to send a text message.
+        /// </summary>
+        /// <param name="client">The DynamoDB client object used to call
+        /// PutItemAsync to add each item to the table.</param>
+        /// <param name="tableName">The name of the table to which the sample
+        /// items will be added.</param>
+        public static async Task AddSampleData(IAmazonDynamoDB client, string tableName)
+        {
+            Table employeeTable = Table.LoadTable(client, tableName);
+
+            // Add sample employee items to the table.
+            // Remember to edit the phone number for emp1 to a valid moble
+            // number that can receive text messages. The hire date is
```

Comment:
do you want to mention how to validate a number? i.e. it has to be registered with SNS
of course, it the user will find this out via the readme or documentation else where there's no need